instrumentation/lettuce/lettuce-5.0/lettuce-5.0.gradle

```diff
@@ -28,7 +28,7 @@ dependencies {
 
   testCompile group: 'com.github.kstyrc', name: 'embedded-redis', version: '0.6'
   testCompile group: 'io.lettuce', name: 'lettuce-core', version: '5.0.0.RELEASE'
+  testCompile project(':instrumentation:reactor-3.1')
```

Comment:
Is this needed for all tests or for `latestDepTest` only?

Reply:
All, the earliest versions of Lettuce 5 use project reactor